### PR TITLE
Added contribution info

### PR DESCRIPTION
**Pull request by hmitsch**
Instead of PRs, please edit the Mozilla Spreadsheet.

### CODE REVIEW

**Comment by hmitsch** [comment]
FYI @MichaelKohler

**Comment by hmitsch** [comment]
@sanacl please review and merge

File: README.md [hunk 1]
```diff
@@ -2,12 +2,15 @@
...
         o   ^__^
          o  (oo)\_______
             (__)\       )\/\
                 ||----w |
                 ||     ||
 ```
+# How to Contribute
+* The list is generated from this spreadsheet: https://docs.google.com/spreadsheets/d/1xFrt9JZkZ6m8peasYaRfPO5kC1JDo-vDJJuYfMwcGqM/edit
+* Please do not submit PRs here but instead edit/comment the spreadsheet. 
+* Ping us on the Community Analytics Telegram channel to trigger a refresh of the list
```
Comment (MichaelKohler) [inline README.md]: Should we post an invite link for that group here? If not, can you add a description on how to get invited to that group?

**Review by MichaelKohler — APPROVED** [review]
Looks good to me, don't have the final say here though. See my comment regarding TG group.

**Comment by canasdiaz** [comment]
@hmitsch @MichaelKohler LGTM.

**Comment by hmitsch** [comment]
@MichaelKohler I addressed your feedback. @sanacl please merge.

**Review by MichaelKohler — APPROVED** [review]
LGTM

**Pull request merged**